### PR TITLE
Updated locators as per changes in https://bugzilla.mozilla.org/show_bug.cgi?id=1254083 and Xfailed tests in prod

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

**Comment by viorelaioia** [comment]
@johngian , can you please review this? thanks!

File: pages/settings.py [hunk 1]
```diff
@@ -222,8 +222,8 @@ def click_find_group_link(self):
 
     class ExternalAccountsTab(PageRegion):
 
-        _external_accounts_form_locator = (By.CSS_SELECTOR, '#extaccounts-tab > div:nth-child(1)')
-        _irc_form_locator = (By.CSS_SELECTOR, '#extaccounts-tab > div:nth-child(2)')
+        _external_accounts_form_locator = (By.CSS_SELECTOR, '#extaccounts>form>div:nth-child(2)')
+        _irc_form_locator = (By.CSS_SELECTOR, '#extaccounts>form>div:nth-child(3)')
```
Comment (akatsoulas) [inline pages/settings.py]: nit: Please add a space between `>` symbols

**Comment by akatsoulas** [comment]
Please rebase this to the master branch

**Comment by viorelaioia** [comment]
Commits were squashed in https://github.com/mozilla/mozillians-tests/pull/274

**Pull request closed without merging**